### PR TITLE
Bugfix/apm 144 gps2 low sats alert

### PR DESCRIPTION
**Pull request by zalice**
Fixed sats min alerts to report both GPS sources if both are below threshold. The last populated prearm message is reported so cannot write the messages independently.

### CODE REVIEW

File: libraries/AP_NavEKF2/AP_NavEKF2_VehicleStatus.cpp [hunk 1]
```diff
@@ -201,17 +201,29 @@ void NavEKF2_core::calcGpsGoodToAlign(void)
...
+                hal.util->snprintf(prearm_fail_string, sizeof(prearm_fail_string),
+                                "GPS1 numsats %u GPS2 %u (need %u)", gps.num_sats(0), gps.num_sats(1), gps.num_sats_arm_min());
+            }
+            else {
+                hal.util->snprintf(prearm_fail_string, sizeof(prearm_fail_string),
+                                "GPS2 numsats %u (needs %u)", gps.num_sats(1), gps.num_sats_arm_min());
+            }
+        }
+        else {
+            hal.util->snprintf(prearm_fail_string, sizeof(prearm_fail_string), "GPS 2 not present for sat check");
```
Comment (adriennet823) [inline libraries/AP_NavEKF2/AP_NavEKF2_VehicleStatus.cpp]: Please remove the space between `GPS` and `2`

File: libraries/AP_NavEKF3/AP_NavEKF3_VehicleStatus.cpp [hunk 2]
```diff
@@ -198,17 +198,29 @@ void NavEKF3_core::calcGpsGoodToAlign(void)
...
+                hal.util->snprintf(prearm_fail_string, sizeof(prearm_fail_string),
+                                "GPS1 numsats %u GPS2 %u (need %u)", gps.num_sats(0), gps.num_sats(1), gps.num_sats_arm_min());
+            }
+            else {
+                hal.util->snprintf(prearm_fail_string, sizeof(prearm_fail_string),
+                                "GPS2 numsats %u (needs %u)", gps.num_sats(1), gps.num_sats_arm_min());
+            }
+        }
+        else {
+            hal.util->snprintf(prearm_fail_string, sizeof(prearm_fail_string), "GPS 2 not present for sat check");
```
Comment (adriennet823) [inline libraries/AP_NavEKF3/AP_NavEKF3_VehicleStatus.cpp]: Same comment as above

**Review by adriennet823 — APPROVED** [review]
One nitpick

File: libraries/AP_NavEKF3/AP_NavEKF3_VehicleStatus.cpp (hunk 2 above)
Comment (zalice) [reply libraries/AP_NavEKF3/AP_NavEKF3_VehicleStatus.cpp]: Done

File: libraries/AP_NavEKF2/AP_NavEKF2_VehicleStatus.cpp (hunk 1 above)
Comment (zalice) [reply libraries/AP_NavEKF2/AP_NavEKF2_VehicleStatus.cpp]: Done!

**Pull request merged**